### PR TITLE
Add another regex to match inline img tags using single quotes

### PR DESCRIPTION
**Pull request by Cadair**
Signed-off-by: Stuart Mumford <stuart@cadair.com>

### CODE REVIEW

File: src/timeline/TimelineModel.cpp [hunk 1]
```diff
@@ -532,6 +532,10 @@ TimelineModel::data(const mtx::events::collections::TimelineEvents &event, int r
                 const static QRegularExpression matchImgUri(
                   "(<img [^>]*)src=\"mxc://([^\"]*)\"([^>]*>)");
                 formattedBody_.replace(matchImgUri, "\\1 src=\"image://mxcImage/\\2\"\\3");
+                // Same regex but for single quotes around the src
+                const static QRegularExpression matchImgUri2(
+                                                            "(<img [^>]*)src=\'mxc://([^\"]*)\'([^>]*>)");
```
Comment (deepbluev7) [inline src/timeline/TimelineModel.cpp:537]: You forgot to change `[^\"]` to single quotes .-.

Comment (Cadair) [reply src/timeline/TimelineModel.cpp:537]:
```suggestion
                                                            "(<img [^>]*)src=\'mxc://([^\']*)\'([^>]*>)");
```

File: src/timeline/TimelineModel.cpp [hunk 2]
```diff
@@ -532,6 +532,10 @@ TimelineModel::data(const mtx::events::collections::TimelineEvents &event, int r
                 const static QRegularExpression matchImgUri(
                   "(<img [^>]*)src=\"mxc://([^\"]*)\"([^>]*>)");
                 formattedBody_.replace(matchImgUri, "\\1 src=\"image://mxcImage/\\2\"\\3");
+                // Same regex but for single quotes around the src
+                const static QRegularExpression matchImgUri2(
+                                                            "(<img [^>]*)src=\'mxc://([^\']*)\'([^>]*>)");
```
Comment (Cadair) [inline src/timeline/TimelineModel.cpp:537]:
```suggestion
                  "(<img [^>]*)src=\'mxc://([^\']*)\'([^>]*>)");
```

**Comment by deepbluev7** [comment]
Thank you!

**Pull request merged**